<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogFLC</class>
 <widget class="QDialog" name="DialogFLC">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>270</width>
    <height>190</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>32</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>270</width>
    <height>190</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>270</width>
    <height>190</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
    <weight>50</weight>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>File location calculator</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QCheckBoxX" name="checkBoxAddressAsHex">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>Values as HEX</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>13</width>
           <height>16</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="labelBaseAddress">
         <property name="text">
          <string>Base address:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="1">
      <widget class="QLineEditX" name="lineEditBaseAddress">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>22</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>22</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>13</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="0">
      <widget class="QRadioButton" name="radioButtonFileOffset">
       <property name="text">
        <string>File offset:</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEditX" name="lineEditFileOffset">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>22</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>22</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButtonX" name="pushButtonFileOffset">
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>22</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>22</height>
        </size>
       </property>
       <property name="text">
        <string notr="true">&gt;</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QRadioButton" name="radioButtonVirtualAddress">
       <property name="text">
        <string>Virt address:</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEditX" name="lineEditVirtualAddress">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>22</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>22</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QRadioButton" name="radioButtonRVA">
       <property name="text">
        <string>Rel virt addr:</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEditX" name="lineEditRVA">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>22</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>22</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButtonX" name="pushButtonOK">
       <property name="minimumSize">
        <size>
         <width>78</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>78</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QPushButtonX</class>
   <extends>QPushButton</extends>
   <header>qpushbuttonx.h</header>
  </customwidget>
  <customwidget>
   <class>QLineEditX</class>
   <extends>QLineEdit</extends>
   <header>qlineeditx.h</header>
  </customwidget>
  <customwidget>
   <class>QCheckBoxX</class>
   <extends>QCheckBox</extends>
   <header>qcheckboxx.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
